mods/ra/fluent/rules.ftl

```diff
@@ -857,8 +857,7 @@ actor-ftrk =
 actor-dtrk =
     .name = Demolition Truck
     .description =
-    Truck carrying armed nuclear explosives,
-    with very weak armor.
+    Truck with actively armed nuclear explosives. Has very weak armor.
```

Comment:
I prefer the other version. Also this sentence would need a new line, it's too long